extensions/analyticsdx-vscode-templates/schemas/template-info-schema.json

```diff
@@ -281,8 +281,7 @@
     },
     "autoInstallDefinition": {
       "type": ["null", "string"],
-      "description": "(Currenly not publically available) Path to the auto install configuration file. Only available for app and embeddedapp templates.",
-      "doNotSuggest": true,
+      "description": "Path to the auto install configuration file. Only available for app and embeddedapp templates. Available in Winter '21 release.",
```

Comment:
@mblumreich The descriptions in the json schema files generally show as hover text in the editor.

Reply:
Got it.
Can we change "Only available for app and embeddedapp templates" to "Only available for app and embeddedapp template types"?

Reply:
```suggestion
      "description": "Path to the auto install configuration file. Only available for app and embeddedapp template types. Available in Winter '21 release.",
```